packages/site/src/components/PaymasterDeployer.tsx

```diff
@@ -0,0 +1,148 @@
+import { ethers } from 'ethers';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+import type { ChainConfigs } from './ChainConfig';
+import { ErrorContainer } from './ErrorContainer';
+import { SuccessContainer } from './SuccessContainer';
+import { TextField } from './TextField';
+import verifyingPaymaster from '../../../snap/artifacts/contracts/samples/VerifyingPaymaster.sol/VerifyingPaymaster.json';
```

Comment:
This requires to build the snap before the site, right? Or are we planning on committing this `.json` file to repo too?

Will this file change from time to time?

Reply:
This file won't be changing, and we do have a post install that would create this artifact after install the dependencies.